tests/feature_extraction/test_feature_calculations.py

```diff
@@ -505,3 +507,26 @@ def test_approximate_entropy(self):
         self.assertEqualOnAllArrayTypes(approximate_entropy, [1, 2, 3], 0, m=2, r=0.5)
         self.assertAlmostEqualOnAllArrayTypes(approximate_entropy, [12, 13, 15, 16, 17]*10, 0.282456191, m=2, r=0.9)
         self.assertRaises(ValueError, approximate_entropy, x=[12, 13, 15, 16, 17]*10, m=2, r=-0.5)
+
+        
+    def test_max_fixed_point(self):
+        """
+        Estimating the intrinsic velocity of a dissipative soliton
+        """
+        settings = FeatureExtractionSettings()
+        default_params = settings.name_to_param['max_fixed_point'][0]
```

Comment:
maybe better just explicitely set the parameters here. Because otherwise it is a but unrelated, that changing the settings object leads to a failing test of a feature calculator... :-)